abci/client/local_client.go

```diff
@@ -4,10 +4,9 @@ import (
 	types "github.com/tendermint/tendermint/abci/types"
 	"github.com/tendermint/tendermint/libs/service"
 	tmsync "github.com/tendermint/tendermint/libs/sync"
+	"golang.org/x/net/context"
```

Comment:
Nit: we should use the std lib context.
```
As of Go 1.7 this package is available in the standard library under the name context. https://golang.org/pkg/context. 
```
(https://pkg.go.dev/golang.org/x/net/context)

Reply:
weird... thanks for noticing!